QtCollider/widgets/QcWebView.h

```diff
@@ -140,19 +135,20 @@ public Q_SLOTS:
     inline static QUrl urlFromString(const QString& str) { return QUrl::fromUserInput(str); }
 
 protected:
-    virtual void keyPressEvent(QKeyEvent*);
     virtual void contextMenuEvent(QContextMenuEvent*);
+    virtual bool event(QEvent* ev);
+    virtual bool eventFilter(QObject* obj, QEvent* event);
```

Comment:
i believe all 3 of these should be marked `override`

Reply:
how about virtual then? shall we keep it or remove it?

Reply:
oh good question. usually i remove virtual when i add override since the virtual part is redundant